velox/dwio/parquet/reader/ParquetReaderUtil.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/dwio/common/compression/Compression.h"
+#include "velox/dwio/parquet/thrift/ParquetThriftTypes.h"
+
+namespace facebook::velox::parquet {
+common::CompressionKind thriftCodecToCompressionKind(
+    thrift::CompressionCodec::type codec);
+} // namespace facebook::velox::parquet
```

Comment:
Please add a newline at the end.

Reply:
Isn't this picked up by the `format-fix` script?

Reply:
Also added an extra line , but the format-check fails.